app/overlays/tgl/ipc4_overlay.conf

```diff
@@ -13,9 +13,9 @@ CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S16_C32=y
 
 # route SOF logs to Zephyr logging subsystem
 CONFIG_SOF_LOG_LEVEL_INF=y
-CONFIG_ZEPHYR_LOG=y
+CONFIG_ZEPHYR_LOG=n
```

Comment:
FYI.
This kconfig option doesn't disable zephyr log, it just stops sof using zephyr logging subsys, log from zephyr os still gets printed. 

CONFIG_LOG=y, this option disables the log subsys totally.

https://github.com/thesofproject/sof/blob/72ad230056002c8122650e872d433002fc8704be/app/prj.conf#L5

Reply:
Ack @aiChaoSONG . To reduce the system load of logging, this is still pretty good as Zephyr doesn't print many logs after boot, so it doesn't add much overhead to SOF execution. And by leaving Zephyr logs enabled, you still get OS panic dumps in mtrace.